libdebug/debugger/internal_debugger.py

```diff
@@ -189,14 +197,14 @@ def clear(self: InternalDebugger) -> None:
 
     def start_up(self: InternalDebugger) -> None:
         """Starts up the context."""
-
         # The context is linked to itself
         link_to_internal_debugger(self, self)
 
         self.start_processing_thread()
         with extend_internal_debugger(self):
             self.debugging_interface = provide_debugging_interface()
-            self.memory = MemoryView(self._peek_memory, self._poke_memory)
+            self._fast_memory = DirectMemoryView(self._fast_read_memory, self._fast_write_memory)
+            self._slow_memory = ChunkedMemoryView(self._peek_memory, self._poke_memory)
```

Comment:
At a quick glance, they are the only two attributes not defined at the top of the file.

Reply:
fixed here [0c06917](https://github.com/libdebug/libdebug/pull/92/commits/0c069175cd956be8875be5711df0d1246a6d17b2)